CONTRIBUTING.md

```diff
@@ -78,9 +78,13 @@ defined at <https://developer.github.com/v3/repos/hooks/> live in
 
 (These notes are mostly only for people merging in pull requests.)
 
-**First and foremost, try to maintain a clean, linear git history.**  With very
-few exceptions, running `git log` should not show a bunch of branching and
-merging.
+**Verify CLAs.** CLAs must be on file for the pull request submitter and commit
+author(s). Google's CLA verification system should handle this automatically
+and will set commit statuses as appopriate. If there's ever any question about
+a pull request, ask [willnorris](https://github.com/willnorris).
+
+**Always try to maintain a clean, linear git history.** With very few
+exceptions, running `git log` should not show a bunch of branching and merging.
 
 Never use the GitHub "merge" button, since it always creates a merge commit.
```

Comment:
I thought about that as I was making this change, but decided not to
include it here in order to keep this focused just on the contributors file.
On Jul 13, 2016 12:26 PM, "Dmitri Shuralyov" <notifications@github.com>
wrote:

> In CONTRIBUTING.md
> <https://github.com/google/go-github/pull/400#discussion_r70681319>:
>
> > @@ -78,9 +78,13 @@ defined at <https://developer.github.com/v3/repos/hooks/> live in
> >
> >  (These notes are mostly only for people merging in pull requests.)
> >
> > -**First and foremost, try to maintain a clean, linear git history.**  With very
> > -few exceptions, running `git log` should not show a bunch of branching and
> > -merging.
> > +**Verify CLAs.** CLAs must be on file for the pull request submitter and commit
> > +author(s). Google's CLA verification system should handle this automatically
> > +and will set commit statuses as appopriate. If there's ever any question about
> > +a pull request, ask [willnorris](https://github.com/willnorris).
> > +
> > +**Always try to maintain a clean, linear git history.** With very few
> > +exceptions, running `git log` should not show a bunch of branching and merging.
> >
> >  Never use the GitHub "merge" button, since it always creates a merge commit.
>
> Maybe this statement should be updated/revisited now that you can merge
> using "rebase and squash" strategy, which means the "since it always
> creates a merge commit" statement is no longer correct.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/google/go-github/pull/400/files/04419bc06b471bbc2e9521fb2e27595141a6a1be#r70681319>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAAEWLM4kJC5C-jw9RYEwQR6jU_H7FF-ks5qVS25gaJpZM4JLq5j>
> .
>